lib/facter/r10k_path.rb

```diff
@@ -0,0 +1,11 @@
+Facter.add(:r10k_path) do
+  confine :kernel => :linux
+  setcode do
+    path = Facter::Util::Resolution.exec("which r10k")
```

Comment:
There are a few ruby examples of doing which so we don't then rely on this bindary, any interest in doing that to make it more platform independent?